scripts/test-coverage-xml.sh

```diff
@@ -18,14 +22,23 @@ export EVIDENTLY_DISABLE_TELEMETRY=1
 
 # The '-vv' flag enables pytest-clarity output when tests fail.
 if [ -n "$1" ]; then
-    coverage run -m pytest $TEST_SRC --color=yes -vv --environment $TEST_ENVIRONMENT --no-provision --cleanup-docker
+    if [ "$STORE_DURATIONS" == "store-durations" ]; then
+        coverage run -m pytest $TEST_SRC --color=yes -vv --environment $TEST_ENVIRONMENT --no-provision --cleanup-docker --store-durations --durations-path=.test_durations
+    else
+        coverage run -m pytest $TEST_SRC --color=yes -vv --durations-path=.test_durations --splits=$TEST_SPLITS --group=$TEST_GROUP --environment $TEST_ENVIRONMENT --no-provision --cleanup-docker
+    fi
 else
-    coverage run -m pytest tests/unit --color=yes -vv --environment $TEST_ENVIRONMENT --no-provision
-    coverage run -m pytest tests/integration --color=yes -vv --environment $TEST_ENVIRONMENT --no-provision --cleanup-docker
+    if [ "$STORE_DURATIONS" == "store-durations" ]; then
+        coverage run -m pytest tests/unit --color=yes -vv --environment $TEST_ENVIRONMENT --no-provision --store-durations --durations-path=unit_.test_durations
```

Comment:
@safoinme what is the `unit_.test_durations` path here? If there are different paths, then shouldn't they be referenced differently below?